### PR TITLE
Italian cola

### PR DESCRIPTION
**Pull request by marcobellagente93**
Questions:
running the model eval I got the following result
{
  "dataset_revision": "f8f98e5c4d3059cf1a00c8eb3d70aa271423f636",
  "mteb_dataset_name": "Itacola",
  "mteb_version": "1.6.1",
  "test": {
    "accuracy": 0.47610256410256413,
    "accuracy_stderr": 0.09784615384615383,
    "ap": 0.838917319311731,
    "ap_stderr": 0.007337806891613534,
    "evaluation_time": 4.39,
    "f1": 0.4089033640095776,
    "f1_stderr": 0.057873962414626424,
    "main_score": 0.47610256410256413
  }
}
so 1) where should this go eventually? and 2) accuracy seem to be worse than random. I did run similar tasks currently present and got a similar impression, am I missing something? As a native speaker the task seems 100% valid and the labels are accurate

## Checklist for adding MMTEB dataset

Reason for dataset addition:

This dataset covers the task of language acceptability for the Italian language, currently missing.

- [x] I have tested that the dataset runs with the `mteb` package.
- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb run -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] I have considered the size of the dataset and reduced it if it is too big (2048 examples is typically large enough for most tasks)
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 
- [] I have added points for my submission to the [POINTS.md](https://github.com/embeddings-benchmark/mteb/blob/main/docs/mmteb/points.md) file.

### CODE REVIEW

**Comment by KennethEnevoldsen** [comment]
Hi @marcobellagente93, thanks for this PR!

>  1) where should this go eventually?

The scores should be added to the PR, when running the command it is automatically added to the `results/{model}/` folder. You can simply commit these.

> 2) accuracy seem to be worse than random

Seems like you have only run one model, do you mind running both to see if one is just surprisingly bad or if it is an issue with the dataset?

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+
+class ItalianLinguisticAcceptabilityClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="Itacola",
+        dataset={
+            "path": "gsarti/itacola",
+            "revision": "f8f98e5c4d3059cf1a00c8eb3d70aa271423f636",
+        },
+        description="An Italian Corpus of Linguistic Acceptability taken from linguistic literature with a binary annotation made by the original authors themselves.",
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:14]: re. score: during the development of the Scandinavian embedding benchmark I discovered that almost no model handles Linguistic acceptability tasks well. 

@Muennighoff, @imenelydiaker I am unsure if this is a shortcoming of current embedding models or not a desired properly (we want them to be semantic). I am in two camps about this.

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        reference="https://aclanthology.org/2021.findings-emnlp.250/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
+        main_score="accuracy",
+        date=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:21]: Should be a range: (start, end)

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        form=["written"],
+        domains=["Non-fiction", "Spoken"],
+        dialect=None,
+        task_subtypes=["Linguistic acceptability"],
+        license=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:26]: please specify this one, if it is not available specify "unknown".

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        reference="https://aclanthology.org/2021.findings-emnlp.250/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["ita-Latn"],
+        main_score="accuracy",
+        date=None,
+        form=["written"],
+        domains=["Non-fiction", "Spoken"],
+        dialect=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:24]:
```suggestion
        dialect=[],
```
Assuming no dialect

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        task_subtypes=["Linguistic acceptability"],
+        license=None,
+        socioeconomic_status=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:27]:
```suggestion
        socioeconomic_status=None, 
```
don't we know this? Where is the data derived from?

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    year = "2021",
+    address = "Punta Cana, Dominican Republic",
+    publisher = "Association for Computational Linguistics",
+    url = "https://aclanthology.org/2021.findings-emnlp.250",
+    doi = "10.18653/v1/2021.findings-emnlp.250",
+    pages = "2929--2940"
+}
+        """,
+        n_samples=None,
+        avg_character_length=None,
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:48]: needed

**Review by KennethEnevoldsen — CHANGES_REQUESTED** [review]
Generally looks very good. A few notes on the metadata.

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 1 above)
Comment (marcobellagente93) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:14]: I wasn't exactly sure what to expect so I pretty much ran the linguistic acceptability evals already implemented and got the same results

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 5 above)
Comment (marcobellagente93) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:27]: The data is derived from linguistics textbooks, should that be marked as "high"?

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 2 above)
Comment (marcobellagente93) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:21]: I have the publication date of course but couldn't find any reference to a data collection period, happy to follow through if there's a suggested guideline for these situations

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 5 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:27]: I would say so.

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 2 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:21]: Create a (from, to) for the publication year then (e.g. if it is 2021: `("2021-01-01", "2022-12-31")`

File: mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py (hunk 1 above)
Comment (KennethEnevoldsen) [reply mteb/tasks/Classification/ita/ItalianLinguistAcceptabilityClassification.py:14]: Can you run a naive Italian encoder on it as well (for the Scandinavian languages they seem to perform above chance).

**Comment by KennethEnevoldsen** [comment]
merged into main via. #444 (I also added 6 points here as well)

**Pull request merged**